.github/workflows/uploadbuildtomuuktest.yml

```diff
@@ -0,0 +1,67 @@
+name: Build and Upload to Muuktest
+
+on:
+  push:
+    branches:
+      - release-candidate
+
+jobs:
+  build:
+    runs-on: macos-latest
+
+    env:
+      ARCHIVE_PATH: ${{ github.workspace }}/build/SPMExample.xcarchive
+      EXPORT_PATH: ${{ github.workspace }}/build
+      TEAM_ID: WTQ4Q66FY6
+      PROVISIONING_PROFILE: io.portalhq.ios.SPM-Example1
+      BUNDLE_ID: io.portalhq.ios.SPM-Example
+      IPA_NAME: SPMExample.ipa
+      API_URL: https://11mi38nzh1.execute-api.us-east-2.amazonaws.com/v1/uploadFile
+      API_KEY: eLiCJrCf2R4uHMNpKAOcC1aTQkdFIy006CeWhFDj
```

Comment:
```suggestion
      API_KEY: ${{ secrets.MUUKTEST_API_KEY }}
```

Reply:
Done, Thank you for help @blakeofwilliam  :), I used the MUUKTEST_API_KEY as name here and same for url. I hope this works